### PR TITLE
New kubernetes module dashboards

### PR DESCRIPTION
**Pull request by MichaelKatsoulis**
## What does this PR do?

This PR modifies existing ones and add more out of the box kubernetes module dashboards as part of https://github.com/elastic/beats/issues/31021

## Why is it important?

The Kubernetes dashboards offered so far where using some fields the wrong way and also where missing a lot of important visualisations.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

- Create a kubernetes cluster with kind
- Bring the stack with `elastic-package stack up --version=8.3.0-SNAPSHOT -v -d `
- Run Metricbeat with image  `8.3.0-SNAPSHOT` in kubernetes https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html
-  Create metricbeat-* data view in Kibana
- Locally switch to the PR's branch and build Metricbeat with `mage build `
- Setup the dashboards with `./metricbeat setup --dashboards -E setup.dashboards.directory=./module/kubernetes/_meta/kibana/ -E setup.kibana.host=http://elastic:changeme@localhost:5601 -e  -d "*"`
- Check the dashboards in Kibana
- For the shake of this testing would be nice that the k8s cluster has some pods,deployments, statefulsets, daemonsets, jobs, cronjobs, pv, pvc, services,namespaces

## Related issues

- 

## Screenshots

Dashboards added:
overview-1
overview-2
dashboards pods
pods-1
dashboards nodes
dashboards deployments
dashboards  statefulset
dashboards volumes
dashboards jobs
dashboards cronjobs
dashboards services
dashboards PVC
dashboards PV:PVC

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -121,6 +121,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Extend documentation about `orchestrator.cluster` fields {pull}30518[30518]
 - Enhance Oracle Module: Change tablespace metricset collection period {issue}30948[30948] {pull}31259[#31259]
 - Add orchestrator cluster ECS fields in kubernetes events {pull}31341[31341]
+- Add new Kubernetes module dashboards {pull}31341[31341]
```
Comment (gizas) [inline CHANGELOG.next.asciidoc:124]: Add new Kubernetes module metricbeat dashboards

maybe to change to this desc

File: metricbeat/module/kubernetes/_meta/kibana/7/dashboard/kubernetes-0a672d50-bcb1-11ec-b64f-7dd6e8e82013.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                "type": "lens",
+                "version": "8.3.0-SNAPSHOT"
+            }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat Kubernetes] Cronjobs",
+        "version": 1
+    },
+    "coreMigrationVersion": "8.3.0",
+    "id": "kubernetes-0a672d50-bcb1-11ec-b64f-7dd6e8e82013",
```
Comment (gizas) [inline metricbeat/module/kubernetes/_meta/kibana/7/dashboard/kubernetes-0a672d50-bcb1-11ec-b64f-7dd6e8e82013.json:75]: Can the id change to sth more intuitive in order t understand dashboard file to edit?
I mean that a name kubernetes-cronjob-xxx can help us later on editing specific file.

Think if makes sense to allign filenames with ids

File: metricbeat/module/kubernetes/_meta/kibana/7/lens/kubernetes-32e1cac0-bcba-11ec-b64f-7dd6e8e82013.json [hunk 3]
```diff
@@ -0,0 +1,203 @@
+{
```
Comment (gizas) [inline metricbeat/module/kubernetes/_meta/kibana/7/lens/kubernetes-32e1cac0-bcba-11ec-b64f-7dd6e8e82013.json:1]: So all dashboards created through lens initially and then exported here correct?
That is why we have this folder?

**Review by gizas — APPROVED** [review]
For the aesthitcs of dashboards we can have a group chat and maybe feedback after the comparisn of diffrent dashboards we have seen from other products.

Only for the main dashboard minor thoughts:
- I would move memory tiles under numeric ones
- I would change `Pod reserved metrics ` to a comparison bar (sth more colourfull) that shows directly the health of cluster and make it our central main indicator
- I would try to keep 3 columns

File: metricbeat/module/kubernetes/_meta/kibana/7/dashboard/kubernetes-0a672d50-bcb1-11ec-b64f-7dd6e8e82013.json (hunk 2 above)
Comment (MichaelKatsoulis) [reply metricbeat/module/kubernetes/_meta/kibana/7/dashboard/kubernetes-0a672d50-bcb1-11ec-b64f-7dd6e8e82013.json:75]: These ids are created by Kibana itself and probably have to follow such patterns.

**Review by gsantoro — APPROVED** [review]

**Review by gizas — APPROVED** [review]

**Pull request merged**